shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/resources/ManagedProcessor.java

```diff
@@ -52,6 +53,17 @@ public static Builder fromBuilder() {
         return new Builder();
     }
 
+    public static ManagedProcessor fromDTO(ProcessorDTO processorDTO, String namespace) {
+        return new Builder()
+                .withNamespace(namespace)
+                .withProcessorId(processorDTO.getId())
+                .withBridgeId(processorDTO.getBridgeId())
+                .withCustomerId(processorDTO.getCustomerId())
+                .withDefinition(processorDTO.getFlows())
+                .withProcessorName(processorDTO.getName())
+                .build();
+    }
+
     public static String resolveResourceName(String id) {
         return OB_RESOURCE_NAME_PREFIX + KubernetesResourceUtil.sanitizeName(id);
```

Comment:
`PROCESSOR_NAME_PREFIX_` instead. The `OB_` part is kinda old-school now before we had RHOSE.